### ISSUE TITLE
Wrong cache-key used for caching

### ISSUE THREAD

**Issue by tytan652** [bilelmoussaoui/flatpak-github-actions#80]
While trying to switch to default cache-key in https://github.com/obsproject/obs-studio/pull/5137.
We were trying remove the `cache-key` settings when this issue happen.

An unexpected issue has occurred:
```
Cache Size: ~1123 MB (1177260662 B)
/usr/bin/tar --use-compress-program zstd -d -xf /__w/_temp/a957e912-7502-4f61-a192-d3e489b47e8d/cache.tzst -P -C /__w/obs-studio/obs-studio
Cache restored successfully
Restored cache with key: -x86_64
```

https://github.com/bilelmoussaoui/flatpak-github-actions/blob/9b33ba80aec2fc2652e05f15f19929fa98182889/flatpak-builder/index.js#L209-L212

It seems that `cacheKey` even if empty does not fullfill the condition and generate a wrong cache key which ends up being `-x86_64` in our situation.

**Comment by andyholmes** [comment]
I believe this one was fixed in #48, it just hasn't had a release since that was merged. I've been using `.../flatpak-builder@master` and it seems to work as expected.

**Comment by bilelmoussaoui** [comment]
Closing as already fixed, I will try to do a new release in the upcoming weeks
